paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -76,8 +76,11 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     private val collectingPhone =
         args.formArgs.billingDetailsCollectionConfiguration.phone == CollectionMode.Always
 
-    private val collectingName =
+    private val collectingName = if (args.instantDebits) {
```

Comment:
Do we have a test for this change?

Reply:
Covered by the end-to-end test. This oversight previously only resulted in `defaultName` being set, which didn’t have any effect in Instant Debits.